examples/src/kotlin/org/partiql/examples/CustomProceduresExample.kt

```diff
@@ -26,12 +27,8 @@ import java.math.RoundingMode
 private val ion = IonSystemBuilder.standard().build()
 
 /**
- * A simple custom stored procedure that calculates the moon weight for each crewmate of the given crew, storing the
- * moon weight in the [EvaluationSession] global bindings. This procedure also returns the number of crewmates we
- * calculated the moon weight for, returning -1 if no crew is found.
- *
- * This example demonstrates how to create a custom stored procedure, check argument types, and modify the
- * [EvaluationSession].
```

Comment:
Think we can include more of these comments. Everything but the last part about modifying the `EvaluationSession` seems the same.